pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/services/ReportMutationService.kt

```diff
@@ -55,7 +55,7 @@ class ReportMutationService : CosmosLoader() {
      * @throws ContentException If there is an error with the content format.
      */
     @Throws(BadRequestException::class, ContentException::class, Exception::class)
-    fun upsertReport(input: ReportInput, action: String): Report? {
+    fun upsertReport(action: String, input: ReportInput,): Report? {
```

Comment:
Is there an extra unwanted "comma" after ReportInput?

Reply:
Removed the extra comma.